content/webapp/components/HTMLSerializers/HTMLSerializers.tsx

```diff
@@ -186,7 +186,7 @@ export const dropCapSerializer: JSXFunctionSerializer = (
     type === prismic.RichTextNodeType.paragraph &&
     children[0] !== undefined
   ) {
-    const firstChild = children[0];
+    const firstChild = children[0] as JSX.Element;
```

Comment:
Don't love this but now that element could be anything, including a boolean, a number, a string... suggestions?